apps/sms/js/utils.js

```diff
@@ -473,25 +473,6 @@
       });
       return parsed;
     },
-    /*
-      Using a contact resolver, a function that can looks for contacts,
-      get the format for the dissambiguation.
-
-      Used mainly in activities since they need to pick a contact from just
-      the number.
-
-      In order to workaround facebook contact issue(bug 895817), it should be
-      able to handle the case about phone number without matched contact.
-
-      Phone number comes directly from the activity in the case we call 'pick'
-      from SMS App.
-    */
-    getContactDisplayInfo: function(resolver, phoneNumber, callback) {
```

Comment:
Since you removed method, it would be great if you look for its usages in the code :) Look at [1] and [2].

If for [2] you just need to remove it, for [1] you should replace "getContactDisplayInfo" usage with its content, e.g.:
```
// instead of: 
recipientPromise = Utils.getContactDisplayInfo(findByAddress, number).then(function onData(data) {
  data.source = 'contacts';
  return data;
});

// you'll have:
findByAddress(phoneNumber).then((contacts) => {
	return Object.assign(
		Utils.basicContact(phoneNumber, contacts),
		{ source: 'contacts' }
	);
});
```

Also I see you work on quite old master branch, please rebase you PR branch on upstream master (I assume you do development on dedicated branch e.g. "feature"):

```
git checkout master
// upstream should be main gaia repo, see https://help.github.com/articles/fork-a-repo/ 
git pull upstream master 
// this is your branch with the patch
git checkout feature
git rebase master
// Once you rebased, you should push rebased branch to origin (your fork)
// PR will be automatically updated.
// When you add new changes, you should create separate commit and push it
// to the same branch - PR will be automatically updated as well.
git push -f origin feature
```

[1] https://github.com/mozilla-b2g/gaia/blob/d544c9b2a77a889d87566aff0ca957da5089a49e/apps/sms/views/conversation/js/conversation.js#L697

[2] https://github.com/mozilla-b2g/gaia/blob/d544c9b2a77a889d87566aff0ca957da5089a49e/apps/sms/views/shared/test/unit/mock_utils.js#L18